components/datatypes/marketing/commerce.schema.json

```diff
@@ -40,28 +40,317 @@
           "description": "Removal or removals of a product entry from a product list, for example a product is removed from a shopping cart."
         },
         "xdm:productListReopens": {
+          "title": "Product List Reopens",
           "$ref": "https://ns.adobe.com/xdm/data/measure",
           "description": "A product list that was no longer accessible (abandoned) has been re-activated by the user. Example via a re-marketing activity."
         },
         "xdm:productListViews": {
+          "title": "Product List Views",
           "$ref": "https://ns.adobe.com/xdm/data/measure",
           "description": "View or views of a product-list has occurred."
         },
         "xdm:productViews": {
+          "title": "Product Views",
           "$ref": "https://ns.adobe.com/xdm/data/measure",
           "description": "View or views of a product have occurred."
         },
         "xdm:purchases": {
+          "title": "Purchases",
           "$ref": "https://ns.adobe.com/xdm/data/measure",
           "description": "An order has been accepted. Purchase is the only required action in a commerce conversion. Purchase must have a product list referenced."
         },
         "xdm:saveForLaters": {
+          "title": "Save for Laters",
           "$ref": "https://ns.adobe.com/xdm/data/measure",
           "description": "Product list is saved for future use, for example a product wish list."
         },
         "xdm:inStorePurchase": {
+          "title": "In-store Purchase",
           "$ref": "https://ns.adobe.com/xdm/data/measure",
           "description": "'inStore' purchase is saved for analytics use."
+        },
+        "xdm:activationFee": {
+          "title": "Activation Fee",
+          "$ref": "https://ns.adobe.com/xdm/datatypes/currency",
+          "description": "Activation fee for an order, if applicable."
+        },
+        "xdm:backorderAmount": {
+          "title": "Backorder Amount",
+          "$ref": "https://ns.adobe.com/xdm/datatypes/currency",
+          "description": "Tracks the total value of backordered items in a visitor's shopping cart."
+        },
+        "xdm:productBrand": {
+          "title": "Product Brand",
+          "type": "string",
+          "description": "Tracks the brand of an individual product."
+        },
+        "xdm:cartID": {
+          "title": "Cart ID",
+          "type": "string",
+          "description": "If applicable, tracks a persistent cart identifier string that would expire upon a successful order event."
+        },
+        "xdm:category": {
```

Comment:
category is too generic. Should we rename this to "productCategory" like "productBrand" ? Also, Do we have to add these properties to the event feed ? Don't they belong to the Product Class and its MIXINS ? We already have a ProductCategory mixin.

Reply:
I've updated these.  For the legacy commerce mixin, we will likely need these as some clients will pass them through in the experience event rather than the classification.  Mitch's forthcoming updates might address some of that.